### PR TITLE
Removing the console and an update on the css

### PR DESCRIPTION
**Pull request by KaayGee**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -69,7 +69,7 @@
     display: block !important;
     padding-top: 19px;
     padding-bottom: 19px;
-    outline: none;
+    text-decoration: none;
```
Comment (paradisekelechi) [inline css/styles.css:72]: Looks good to me

File: index.html [hunk 2]
```diff
@@ -132,11 +132,6 @@ <h2>Wedding Events</h2>
         </div>
     </section>
 
-    <script>
```
Comment (paradisekelechi) [inline index.html:135]: Why did you remove these? @KaayGee

File: css/styles.css (hunk 1 above)
Comment (KaayGee) [reply css/styles.css:72]: Thank you

File: index.html (hunk 2 above)
Comment (KaayGee) [reply index.html:135]: It wasn't necessary.
